src/components/ObjectArrayEditor.vue

```diff
@@ -137,12 +144,23 @@ const TableCom = ref()
 
 const { initRecordByComponents } = useSchemaRecord()
 
+const initRecordByPluginForm = (data: Properties) => {
+  return Object.keys(data)
+    .map((key) => ({ [key]: '' }))
+    .reduce((acc, cur) => ({ ...acc, ...cur }), {})
+}
+
 const keyArr = computed(() => Array.from({ length: arr.value.length }, () => createRandomString()))
 
 const getColumnWidth = (property: Property) => (property.type === 'object' ? 300 : undefined)
 
 const addItem = () => {
-  const objData = get(initRecordByComponents(props.properties), props.propKey)
+  let objData
+  if (props.inPlugins) {
+    objData = initRecordByPluginForm(props.properties)
+  } else {
+    objData = get(initRecordByComponents(props.properties), props.propKey)
+  }
```

Comment:
I don't have a good idea on how to handle this at the moment, so I've made a special arrangement.

Reply:
LGTM